docs/TemplateManagementCLI.md

```diff
@@ -5,42 +5,44 @@ The Template Management CLI tool is mean to pull, push, and manage the templates
 # Using Template Management CLI
 
-The command-line tool can be used to pull and push a template collection from/to a remote registry (Now we only support Azure Container Registry). 
+The command-line tool can be used to pull and push a template collection from/to Azure Container Registry. For now we only support windows, and will support other OS soon.
 
 ## Prerequisites
 * Azure container registry - Create a container registry in your Azure subscription if you do not have one. This is the registry where you want to keep your Liquid templates. You can use the [Azure portal](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-get-started-portal) or the [Azure CLI](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-get-started-azure-cli).
 
-* Azure Active Directory service principal (optional) - If using service principal's identity for authentication, you need to create a [service principal](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-auth-service-principal) to access your registry. Ensure that the service principal is assigned a role such as AcrPush so that it has permissions to push and pull artifacts.
 
-* Azure CLI (optional) - To use an individual identity, you need a local installation of the Azure CLI. Version 2.0.71 or later is recommended. Run `az --version` to find the version. If you need to install or upgrade, see [Install Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli).
-
-* Docker (optional) - To use an individual identity, you must also have Docker installed locally, to authenticate with the registry. Docker provides packages that easily configure Docker on any [macOS](https://docs.docker.com/docker-for-mac/), [Windows](https://docs.docker.com/docker-for-windows/), or [Linux](https://docs.docker.com/engine/install/) system.
 ## Authentication
 
 Before pull & push operations, azure authentication is required for private registries. Customers can directly use individual login with Azure AD through [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/authenticate-azure-cli) or use identity (individual identity or Azure AD [service principal identity](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-auth-service-principal)) to sign in the registry. 
 
 ### Login Using Azure CLI
 
-After signing in to the Azure CLI with your identity, use the Azure CLI command `az acr login` to access the registry.
+To use individual login with Azure AD, you need a local installation of the Azure CLI. Version 2.0.71 or later is recommended. Run `az --version` to find the version. If you need to install or upgrade, see [Install Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli).
+
+You should first sign in to the Azure CLI with your identity, and then use the Azure CLI command `az acr login` to access the registry.
```

Comment:
Does the user need to have permissions on the registry in order to pull/push? If so, please mention what permission is needed.